Added video/3gpp filetype

apmuthu opened the pull request:
DB Schema & SQL cleanup, fixes

apmuthu committed:
Added video/3gpp filetype

apmuthu committed:
Moved js from footer to header in tweeter template

apmuthu committed:
show_footer boolean setting added

apmuthu committed:
Computed base_url initial value during install

apmuthu committed:
Added Entity Relationship Diagram (ERD)

apmuthu committed:
Removed redundant indexes and SQL cleanup

apmuthu committed:
Updated installer with SQL optimization of previous commit.

apmuthu committed:
Space separator for between "Choose" and Author / Dept / File Category in Filter box

apmuthu committed:
Added Tamil language

apmuthu committed:
Added self to Credits

apmuthu committed:
Removed Google Ads in demo mode in footers

apmuthu committed:
sql cleanup

stephenlawrence commented:
I better way to handle this type of merge is to create individual issues on the opendocman page and then create individual pull requests for each.

apmuthu commented:
@stephenlawrence: Thanks for the comments. I can create separate issues for these changes, but how does the separate pull request work? When I make a pull request, all future changes also get added in till the issue is merged or rejected!

apmuthu commented:
The *install/odm.php* is uptodate with the *database.sql*.

apmuthu committed:
Added Upgrade Script for DB Schema changes till date
Synched database.sql with install/odm.php
Arranged the tables in alphabetical order in database.sql to match dumps
https://github.com/opendocman/opendocman/pull/220#commitcomment-13640922

apmuthu committed:
descending => Descending in translations

apmuthu committed:
Added Install Upgrade sql function and cleanup

apmuthu committed:
Fixed #8 - Category displayed in File Listing
Cleaned up all language files and synced them with english file
All language files are now in same string order

apmuthu committed:
Bumped version tp 1.3.5 for DB schema changes since 1.3.4 was already released

apmuthu committed:
Further SQL cleanups and Primary keys for all tables

stephenlawrence commented:
I believe you would create a branch and odm issue for each of these changes and create a pull request from that branch into odm master.

apmuthu committed:
Updated the Entity Relationship Diagram (ERD) for ODM v1.3.5

apmuthu commented:
Have completed all SQL changes, Upgrades, Language cleanups, ERD updations, Categories in File List display. 

Next time I will make separate unit pull requests. Not proficient with Git branching and merging several tracks as yet.

You can now pull it all in. Just did a successful dry run.

stephenlawrence commented:
While I appreciate the efforts here this is not a good way of submitting code. I cannot merge in a large list of different issues, I need to be able to accept or reject individual changes and make comments on them. Some of them will not be included, some need changes applied, etc.. I have created new Issues for some of your changes which you can create branches for.

apmuthu committed:
Fixed typo in upgrade link

apmuthu committed:
Inducted part of upstream changes in SQL
https://github.com/opendocman/opendocman/commit/c5978c4547a654684cafae09e72cc2c04a02c74b

apmuthu committed:
Smarty assignment of $state and cleanup
https://github.com/opendocman/opendocman/commit/438e84f164dc1b19819a538fe78b9fa06d957c58#diff-e906deb3d2f03e8ea4d0f047103b2a08
https://github.com/opendocman/opendocman/commit/438e84f164dc1b19819a538fe78b9fa06d957c58#commitcomment-15110194

apmuthu committed:
Removed redundancy of $state computation

apmuthu committed:
Syntax fixes from #242
https://github.com/opendocman/opendocman/commit/438e84f164dc1b19819a538fe78b9fa06d957c58#diff-5478d4cef1e0d6ae027543b823b181d4

apmuthu committed:
Fixes #240 - add correct port details to config sample
https://github.com/opendocman/opendocman/commit/27e99a39afb98d4d48880009fe8e70160c11bb66

apmuthu committed:
HTML Beautifier Formatted login.tpl and inserted missing </p> tag

apmuthu committed:
Fixed #245 - revision view not working for historical revisions
https://github.com/opendocman/opendocman/commit/6e03c226bcd8cfbe695ed583c6ed802911ce571d

apmuthu committed:
Whitespace fixes

apmuthu committed:
Matched upstream variable naming: $installpath => $computed_base_url

apmuthu committed:
Fixed #244 - password fields during install
https://github.com/opendocman/opendocman/commit/8a412036f9f0e8849a1300224d578b0029fd9d51

apmuthu committed:
Input Validation
https://github.com/opendocman/opendocman/commit/08927e0679a85215d879e939a4fb90e8db386e8d#diff-64fbc1aff8aee6beb405eb70760e442a

apmuthu committed:
Updated database.sql for newer MySQL versions min date fix
https://github.com/opendocman/opendocman/commit/701d219e6852f096cf04548dabc3b46a68eac600
The new minimum value is 1000-01-01 00:00:00
https://dev.mysql.com/doc/refman/5.5/en/datetime.html

apmuthu committed:
Fix #257 - space in dutch lang file causing file errors
https://github.com/opendocman/opendocman/commit/1dcd9998254b5ed7ea4a903ddd2627f94bbade7e

apmuthu committed:
Fixed #263 - Move header above form
https://github.com/opendocman/opendocman/commit/331f2556d8da91361d6c19ede4d5f57d99944cd9

apmuthu committed:
Fixes #117 deleting user leaves files in limbo
https://github.com/opendocman/opendocman/commit/3cfd5aea6e433aaa90991a78e00d5e7c8f2e52cc

apmuthu committed:
Added dummy docs/index.html
https://github.com/opendocman/opendocman/commit/574f6b3aaf80e374f466568ca7de8b11ed6caf81

apmuthu committed:
Fixed #270 PHP ROUNDing errors
https://github.com/opendocman/opendocman/issues/270

apmuthu committed:
Added *.mid mime type - audio/midi
